### ISSUE TITLE
Conclusion should be backmatter

### ISSUE THREAD

**Issue by drgrigg** [vr8hub/mark-twain_the-adventures-of-tom-sawyer#4]
I think the Conclusion, which are words directly from the author, should be a backmatter item.

In any case, it should have the correct id. You have:

`<body epub:type="bodymatter z3998:fiction">
		<section id="chapter-36" epub:type="chapter">`

I think this should be:

`<body epub:type="backmatter z3998:non-fiction">
		<section id="conclusion" epub:type="conclusion">`

**Comment by drgrigg** [comment]
We might need to get Alex's views on the bodymatter/backmatter decision. @acabal ?

**Comment by vr8hub** [comment]
Yep, missed the section id when I changed everything else. (It's always something.) For now, I've changed it to backmatter; I'll be glad to change it back if @acabal prefers it as bodymatter.

**Comment by drgrigg** [comment]
But I also think you need to change 'z3998:fiction' to 'z3998:non-fiction' because it's an address to the reader from the author and not part of the story. The same, actually, applies to the preface:

> Most of the adventures recorded in this book really occurred; one or two were experiences of my own, the rest those of boys who were schoolmates of mine.

**Comment by vr8hub** [comment]
Really? That's … interesting. I haven't seen that in the docs, and I haven't done that before, and no one's said anything (including you, on Notre Dame :) ). Both Notre Dame and Tale of Two Cities should be fixed, then.

Preface/conclusion are both fixed and pushed.

**Comment by drgrigg** [comment]
Having looked at them both now, yes,I think the prefaces of both Notre Dame and Tale of Two Cities should also be tagged non-fiction. It’s pretty clear that these bits aren’t FICTION but FACTUAL.  Given that we go to the trouble of distinguishing real quotes from fictional quotes (see the Semantics Manual under “Blockquotes” I think it’s worth getting this right on these instances.  I might trawl through the whole corpus to see if I can find similar examples. @acabal, do you want to weigh in on this?

**Comment by drgrigg** [comment]
The manuals do definitely need updating and expanding, but Alex is so busy it may not be something that happens soon.

**Comment by vr8hub** [comment]
Oh, it makes total sense once you pointed it out. It just never occurred to me, since there's no mention of it, and it wasn't mentioned on prior books.

Note that Notre Dame has both a preface and a note at the end; both should be non-fiction. If you don't get to it and Tale of Two Cities first, I'll try to do PR's, but it will be a few days before I can get to them.

**Comment by drgrigg** [comment]
No, it’s OK. I can fix them directly. I’ll do it later today.

And I’ll also figure out a way of automating the checking of the whole corpus for similar instances, though each will need manual attention to be sure of the fiction/nonfiction status of files like prefaces, introductions, etc.

**Comment by acabal** [comment]
Correct on fiction vs. nonfiction.

Whether a conclusion is backmatter is a bit of a matter of opinion. Would we consider the conclusion optional? If so then it's backmatter. If not then it would be bodymatter.

**Comment by drgrigg** [comment]
I would say that in this instance it perfectly mirrors the preface, which is frontmatter. You could read the book happily without reading either preface or conclusion, so I’d say it’s optional and so should be backmatter.

**Comment by vr8hub** [comment]
Agreed.
